### PR TITLE
chore(ci): update coderabbit config

### PR DESCRIPTION
**Pull request by swift1337**
```
__________________________________________
< My GPUs are ready. Let's find some bugs. >
------------------------------------------
 \
  \   (\__/)
      (•ㅅ•)
      / 　 づ
```

image

## Summary by CodeRabbit

- **New Features**
	- Enhanced review process with new configuration options for profiles, summaries, and more.
	- Introduced additional filtering for TypeScript declaration files during reviews.
	- Integration of the `languagetool` for improved code quality checks.

- **Improvements**
	- Streamlined review output with the introduction of new parameters in the reviews section.
	- Disabled auto-incremental reviews for more controlled review practices.

### CODE REVIEW

File: .coderabbit.yaml [hunk 1]
```diff
@@ -13,8 +31,17 @@ reviews:
...
       instructions: >-
-          Review the shell scripts, point out issues relative to security,
-          performance, and maintainability.
+        Review the shell scripts, point out issues relative to security,
+        performance, and maintainability.
+
   auto_review:
+    drafts: false
+    # disables review for new commits
+    auto_incremental_review: false
```
Comment (gartnera) [inline .coderabbit.yaml:40]: I think the incremental reviews are good?

File: .coderabbit.yaml [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+chat: { auto_reply: true }
+tone_instructions: >-
+  Maintain a formal tone, highlighting issues, and suggesting production-grade, elegant, and concise solutions.
+
 reviews:
+  profile: chill
+  high_level_summary: true
+
+  # disables the cringe
+  poem: false
```
Comment (gartnera) [inline .coderabbit.yaml:13]: I agree

**Review by gartnera — APPROVED** [review]

File: .coderabbit.yaml (hunk 1 above)
Comment (lumtis) [reply .coderabbit.yaml:40]: Yeah, @swift1337  do you have an example where this is not relevant to keep it?
Can we try first with the new directive in the config to see?

Comment (lumtis) [reply .coderabbit.yaml:40]: Finally approving it. I think we can ask review manually if it is relevant.

**Review by lumtis — APPROVED** [review]

File: .coderabbit.yaml (hunk 1 above)
Comment (swift1337) [reply .coderabbit.yaml:40]: I see a lot of redundant follow up of new commits. Indeed let's ask for incremental reviews manually.

image
image
image


etc...

**Pull request merged**